Resources/ServerInfo/Guidebook/Engineering/TEG.xml

```diff
@@ -21,19 +21,153 @@
 
   There is no preference for which side must be hot or cold, there need only be a difference in temperature between them. The gases in the two "loops" are never mixed, only energy is exchanged between them. The hot side will cool down, the cold side will heat up.
 
-  ## Burn Chamber
+  ## The Pipes
+
+  There are 2 major pipenets to worry about here: The Hot Loop (where gas will be burnt for heat), and The Cold Loop (where circulated, heated waste gas will either be removed into space or cooled back down). Make sure that [bold]both pipenets do NOT mix[/bold], as only heat should be transferred between the two through the TEG.
+
+  # The Hot Loop
+
+  As I'm sure a wise person once said: the best way to make something hot is to light it on fire. Well, depending on context, that may not be very wise, but luckily your engineering department has just what's needed to do it wisely after all.
+
+  As stated above, there are many different layouts one can follow to heat up (or cool down) gases; this part of the guide will cover 2 common methods one will often see for the hot loop when the TEG is setup: The Pipe Burn, and the Burn chamber.
+
+  Side note: Plasma fires burn now relatively cool compared to, for example, Tritium fires (compared to before, where Plasma fires were near-instant levels of lethal). It may be viable to extract Tritium from the setups below (using a 95/5 ratio of O2/Plasma) and react it with Oxygen to get truly hellish temperatures for power. Although, this is just a recommendation; I'm not ya mum.
```

Comment:
98/2 or 97/3

Reply:
i should actually test which one runs hotter, unless they both run at the same temperature because an optimum mix is between these two ratios

Reply:
I have tested both mixes, and I deduce 97/3 has a higher yield (more WV, more trace Trit, higher temp)
Thereby, 97/3 goes into the guide